app/presenters/arclight/show_presenter.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Arclight
+  # Custom presentation methods for show partial
+  class ShowPresenter < Blacklight::ShowPresenter
+    def heading
+      title = super
+      date = document.fetch('unitdate_ssm', [])
+      title += ", #{date.first}" if date.present?
```

Comment:
Is the `if data.present?` needed here? Won't `fetch` return `[]` if its not there?